integration/internal/api/plugin/plugin.go

```diff
@@ -0,0 +1,59 @@
+package plugin
+
+import (
+	"context"
+	"io/ioutil"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+)
+
+// InstallGrantAllPermissions installs the plugin named and grants it
+// all permissions it may require
+func InstallGrantAllPermissions(client client.APIClient, name string) error {
```

Comment:
This can be an unexported helper in the one file where it is used.

The rest of the file should be removed, they are one line wrappers around the client.

Reply:
This will be used by other test suites -- at least mount point plugins. The other functions in this file exist to provide 1:1 mapping to the command line without depending on `docker/cli`. The client library imposes significant incidental complexity (contexts, options, questionable semantic decisions like in `PluginInstall`, etc) that a developer writing or porting a test shouldn't have to be subjected to. If, eventually, there is actually no benefit to fixing up the `APIClient` interface for test suite users, great. Until that time, this design makes it much easier to work with by e.g. not requiring questions like "Is there a utility function for this or is this one of the annoying `APIClient` functions?"

Reply:
> 1:1 mapping to the command line

This is exactly what we do not want. If we were going to provide a 1:1 mapping to the CLI there would be little value in removing the CLI from the test suite.

If there are helpers that will be shared between the plugin suite then maybe create an `integration/plugin/internal` ?